docs/docs/how-to/performance/improving-site-performance.md

```diff
@@ -93,21 +95,29 @@ To reduce your JavaScript bundle size, follow the steps below.
 
 #### Step 1: Profile your bundle
 
-The first step to fix this is to figure out what's going on. Use `gatsby-plugin-webpack-bundle-analyzer` to profile your bundle. When you add this tool to your `gatsby-config.js`, you can analyze the bundle size on each page in your application.
+The first step to fix this is to figure out what's going on. Use `gatsby-plugin-webpack-bundle-analyzer` and the _experimental_ plugin `gatsby-plugin-perf-budgets` to profile your bundle. When you add this tool to your `gatsby-config.js`, you can analyze the bundle size on each page in your application.
+
+Chunk naming patterns help you break down a page into three categories:
+
+- application-level chunks (shared with all pages),
+- template-level chunks (shared with all pages of the same template),
+- page-level data imports
 
-#### Step 2: Remove unneeded third-party imports from `commons.js`
+Visuals illustrating this breakdown: [all chunks on a page](https://docs.google.com/presentation/d/1tRsmG77NqG85tTbNZF9lDIKe44NPNqZgzWA-J_seQek/edit#slide=id.gbd2ab6a11d_0_35), [only application-level chunks](https://docs.google.com/presentation/d/1tRsmG77NqG85tTbNZF9lDIKe44NPNqZgzWA-J_seQek/edit#slide=id.gbd2ab6a11d_0_29), [only template-level chunks](https://docs.google.com/presentation/d/1tRsmG77NqG85tTbNZF9lDIKe44NPNqZgzWA-J_seQek/edit#slide=id.gbd2e5f47fd_0_11).
```

Comment:
Better link images here instead of slides (someone might delete/move/change permissions on that slide deck)

Reply:
Good point. Adding better URL.